resources/skins/Main/1080i/script-plex-pre_play.xml

```diff
@@ -256,6 +257,53 @@
                         <label>$INFO[Window.Property(remainingTime)]</label>
                     </control>
                 </control>
+
+                <control type="grouplist">
```

Comment:
What's this?

Reply:
It's just shifting the control group code from line 324 to 261 so it's right after the control group for the title/edition stuff (to be easier to keep track of going forward with the ratings moved up to the same line as those items).

Reply:
FYI it does also include the tweaks to alignment/pixel values for the control group.

Reply:
Oh OK. I haven't tested those. Please reopen the PR if you want me to re-check :)

Reply:
Would it be easier for now if I modified my fork, to keep that control group at the original line number so just the key changes are highlighted in my commit?

Reply:
No idea to be honest.